### ISSUE TITLE
train list and test list

### ISSUE THREAD

**Issue by FrenchKiwiJuice** [Jia-Research-Lab/PFENet#3]
Can you share the train_list.txt and val_list.txt file? I'd like to know the size and division of your training set and test set

**Comment by tianzhuotao** [comment]
Hi,

We have accordingly uploaded the lists and updated README.md.

Thank you.
